RecoTracker/TkTrackingRegions/src/OuterHitPhiPrediction.h

```diff
@@ -4,32 +4,40 @@
 /** predicts phi range at a given radius r */
 
 #include "RecoTracker/TkMSParametrization/interface/PixelRecoRange.h"
-#include "RecoTracker/TkTrackingRegions/interface/TkTrackingRegionsMargin.h"
 
 #include "FWCore/Utilities/interface/Visibility.h"
 
+#include <cassert>
+
 class dso_internal OuterHitPhiPrediction {
 public:
 
-  typedef PixelRecoRange<float> Range;
-  typedef TkTrackingRegionsMargin<float> Margin;
+  using Range= PixelRecoRange<float>;
 
   OuterHitPhiPrediction( 
       const Range & phiAtVertex, 
       const Range & curvature, 
-      float originRBound, 
-      const Margin & tolerance = Margin(0,0))
+      float originRBound) 
     : thePhiAtVertex(phiAtVertex), theCurvature(curvature),
-      theOriginRBound (originRBound), theTolerance(tolerance) { } 
+      theOriginRBound (originRBound) {
+       assert(theCurvature.max()>0);
```

Comment:
let's clear your confusion then :) 
1) I definitely don't want to introduce open snapshot GTs (such as 80X_dataRun2_HLT_v*), because this (if used by mistake in any production like workflow) will result in non-reproducibilities.
2) The PR you are referring to is not yet merged
3) RelVals should definitely use the `auto:run2_hlt_relval`as they are correctly doing.
4) the one and only reason why I haven't cleaned up yet the key `'auto:run2_hlt` from autoCond is because TSG is using it all over the place (https://github.com/cms-sw/cmssw/search?utf8=%E2%9C%93&q=run2_hlt).
5) producing a new GT (actually same GT, but with newer snapshot time)  for the `80X_dataRun2_HLT_frozen_v`can be done (we are in any case planning to make a PR to add new records in 81x soon).

In summary:
   * using `auto:run2_hlt_relval` instead of `auto:run2_hlt` is up to you, depending if you need the L1 menu from O2O or a fixed version (the latest) available in the `auto:run2_hlt_relval` key.
   * the fact that in this particular case you've got wrong conditions in one case in bad conditions in another is just an accident due to the particular time of the snapshot. In both case is still **wrong** to reconstuct data taken past the snapshot